my-app/src/main/java/com/JavaWebServer/app/RESTMethods/Get.java

```diff
@@ -16,12 +16,11 @@
   private String statusError = "HTTP/1.1 500 Internal Server Error";
   private String fileName;
   private String CRLF ="\r\n";
```

Comment:
If this is a constant, which the naming leads me to believe it is, it should be `final` and `static` as well.